modules/kubernetes/aks-core/k8s-cluster-role.tf

```diff
@@ -20,24 +26,30 @@ resource "kubernetes_cluster_role" "helm_release" {
   }
 }
 
-resource "kubernetes_cluster_role" "toolkit_helm_release" {
+resource "kubernetes_cluster_role" "toolkit_edit" {
   metadata {
-    name = "toolkit-helm-release"
+    name = "toolkit-edit"
+    labels = {
+      "xkf.xenit.io/kind" = "platform"
+    }
   }
   rule {
-    api_groups = ["helm.toolkit.fluxcd.io"]
+    api_groups = ["helm.toolkit.fluxcd.io", "kustomize.toolkit.fluxcd.io", "source.toolkit.fluxcd.io", "notification.toolkit.fluxcd.io"]
```

Comment:
Multi-line formatting would be easier to read, i think.

Reply:
Done